src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceTransformer.java

```diff
@@ -0,0 +1,839 @@
+import com.google.api.codegen.transformer.PathTemplateTransformer;
+import com.google.api.codegen.transformer.RetryDefinitionsTransformer;
+import com.google.api.codegen.transformer.ServiceTransformer;
+import com.google.api.codegen.transformer.StandardImportSectionTransformer;
+import com.google.api.codegen.transformer.StaticLangApiMethodTransformer;
+import com.google.api.codegen.transformer.SurfaceNamer;
+import com.google.api.codegen.transformer.SurfaceTransformer;
+import com.google.api.codegen.util.TypeAlias;
+import com.google.api.codegen.viewmodel.ApiCallSettingsView;
+import com.google.api.codegen.viewmodel.ApiMethodView;
+import com.google.api.codegen.viewmodel.ClientMethodType;
+import com.google.api.codegen.viewmodel.ImportSectionView;
+import com.google.api.codegen.viewmodel.PackageInfoView;
+import com.google.api.codegen.viewmodel.PagedResponseIterateMethodView;
+import com.google.api.codegen.viewmodel.ServiceDocView;
+import com.google.api.codegen.viewmodel.SettingsDocView;
+import com.google.api.codegen.viewmodel.StaticLangApiMethodView;
+import com.google.api.codegen.viewmodel.StaticLangApiView;
+import com.google.api.codegen.viewmodel.StaticLangFileView;
+import com.google.api.codegen.viewmodel.StaticLangPagedResponseView;
+import com.google.api.codegen.viewmodel.StaticLangPagedResponseWrappersView;
+import com.google.api.codegen.viewmodel.StaticLangRpcStubView;
+import com.google.api.codegen.viewmodel.StaticLangSettingsView;
+import com.google.api.codegen.viewmodel.StaticLangStubInterfaceView;
+import com.google.api.codegen.viewmodel.ViewModel;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/** A transformer to transform an ApiModel into the standard GAPIC surface in Java. */
+public class JavaSurfaceTransformer {
+  private final GapicCodePathMapper pathMapper;
+  private final PackageMetadataConfig packageMetadataConfig;
+  private final SurfaceTransformer surfaceTransformer;
+
+  private final ServiceTransformer serviceTransformer = new ServiceTransformer();
+  private final PathTemplateTransformer pathTemplateTransformer = new PathTemplateTransformer();
+  private final ApiCallableTransformer apiCallableTransformer = new ApiCallableTransformer();
+  private final StaticLangApiMethodTransformer apiMethodTransformer =
+      new StaticLangApiMethodTransformer();
+  private final PageStreamingTransformer pageStreamingTransformer = new PageStreamingTransformer();
+  private final BatchingTransformer batchingTransformer = new BatchingTransformer();
+  private final StandardImportSectionTransformer importSectionTransformer =
+      new StandardImportSectionTransformer();
+  private final FileHeaderTransformer fileHeaderTransformer =
+      new FileHeaderTransformer(importSectionTransformer);
+  private final RetryDefinitionsTransformer retryDefinitionsTransformer =
+      new RetryDefinitionsTransformer();
+  private final String RPC_STUB_TEMPLATE_FILENAME;
```

Comment:
This shouldn't use constant-style casing if it is passed in from the constructor.